tests/queries/0_stateless/02884_create_view_with_sql_security_option.sh

```diff
@@ -92,6 +90,7 @@ ${CLICKHOUSE_CLIENT} --user $user2 --query "SELECT count() FROM $db.test_view_10
 
 ${CLICKHOUSE_CLIENT} --query "ALTER TABLE $db.test_view_10 MODIFY SQL SECURITY INVOKER"
 (( $(${CLICKHOUSE_CLIENT} --user $user2 --query "SELECT * FROM $db.test_view_10" 2>&1 | grep -c "Not enough privileges") >= 1 )) && echo "OK" || echo "UNEXPECTED"
+${CLICKHOUSE_CLIENT} --query "SHOW CREATE TABLE $db.test_view_10" | grep -c "SQL SECURITY INVOKER"
```

Comment:
What will this line show if we run this test without this your fix?

Reply:
0, because ALTER hadn't overridden the table's DDL properly. I missed it in the test because the table itself behaved correctly (see test on the line above), but all `ALTER`'s changes were lost after restart.